src/main.cpp

```diff
@@ -13,23 +17,36 @@ int main(int argc, char* argv[])
 	game = new Game();
 
 	game->init("No_Name_Chicken_Game", SDL_WINDOWPOS_CENTERED, SDL_WINDOWPOS_CENTERED, SCREEN_SIZE_WIDTH, SCREEN_SIZE_HEIGHT, false);
-	while (game->running())
-	{
-		frameStart = SDL_GetTicks();
+    while(playing) {
+        while (game->running()) {
+            frameStart = SDL_GetTicks();
+
+            game->handleEvents();
+            game->update();
+            game->render();
+
+            frameTime = SDL_GetTicks() - frameStart;
+
+            if (frameDelay > frameTime) {
+                SDL_Delay(frameDelay - frameTime);
+            }
+        }
+        bool winner = game->getWinner();
 
-		game->handleEvents();
-		game->update();
-		game->render();
+        PopupWindow popupWindow("Game over",winner ? "Player1 won! Press 'C' to continue or 'Q' to quit."
+                                       : "Player2 won! Press 'C' to continue or 'Q' to quit.");
 
-		frameTime = SDL_GetTicks() - frameStart;
+        popupWindow.renderWinnerPopup(winner);
 
-		if (frameDelay > frameTime)
-		{
-			SDL_Delay(frameDelay - frameTime);
-		}
-	}
+        while (!popupWindow.interacted) {
+            popupWindow.handleWinnerEvents();
```

Comment:
maybe instead wait in handleWinnerEvents() for a keyboard input in that function